.github/labeler.json

```diff
@@ -15,6 +15,12 @@
       { "type": "user", "pattern": "kyliau" },
       { "type": "user", "pattern": "spanicker" }
     ],
+    "created-by: Trusted contributors": [
+      { "type": "user", "pattern": "anonrig" },
+      { "type": "user", "pattern": "SukkaW" },
+      { "type": "user", "pattern": "unstubbable" },
+      { "type": "user", "pattern": "vercel-release-bot" }
```

Comment:
Existing users are taken from https://github.com/vercel/nextjs-ci-config/blame/19a99c7faa9a3807d0ec598741de0364cc31a538/validate-job.js#L3 and then I added @unstubbable so that I don't need to babysit CI for him :P